### ISSUE TITLE
Fullscreening a video scrolls feed back to top

### ISSUE THREAD

**Issue by wem1c** [trunks-social/trunks-public#223]
## Issue
While playing a video no problems whatsoever, going into fullscreen and then exiting fullscreen will return your view back to the top of the feed.

## Proof
[*screenrec*](https://imgur.com/a/qw0fe7u)

## Sys info
- Android 13
- MIUI Global 14.0.7
- Trunks ver. 1.2.5:130

**Comment by Decad3nce** [comment]
Yeah, this is currently a pretty gnarly bug. Thanks for the report!

**Comment by maksimovic** [comment]
This also happens if you just rotate your phone to landscape and then back to portrait. I have it on auto mode and I sometimes when I just put the phone on the table it flips to landscape. Grabbing it back and going into portrait scrolls the feed to the top 😭
